router_test.go

```diff
@@ -4,7 +4,7 @@
 
 package fiber
 
-// go test -v ./... -run=^$ -bench=Benchmark_Router_Handler -benchmem -count=3
+// go test -v ./... -run=^$ -bench=Benchmark_Router -benchmem -count=2
```

Comment:
I believe that's interesting we show results about these changes but now we are using other parameters. I referer about add this info on the description of this pull request.

Reply:
is only a comment, what was used internally, so here is no problem

Reply:
@ReneWerner87 I know, but I believe that it is always good to have rich descriptions to better tell the community about the changes, etc.